src/crypto.rs

```diff
@@ -123,6 +169,14 @@ impl NamedKey {
             .insert("e", json_big_num(&e))
             .build()
     }
+
+    // Check if the key is valid at the given time.
+    pub fn is_valid_at(&self, time: &DateTime<UTC>) -> bool {
+        match self.valid_from {
+            Some(ref valid_from) => time >= valid_from,
+            None => true
```

Comment:
Do I understand that right that this will make the key be valid if valid_from is not set?